modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -1,40 +1,51 @@
 
-Software requirements are:
+== Minimum Software Requirements
 
-* One of the following Java versions:
-+
-** Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended)
+Mule requires minimum software resources:
+
+* Supported Java versions:
 +
-[NOTE]
---
-If you are planning to run a Standalone instance of Mule Runtime Anypoint Studio freezes during start up if your installed JDK runtime is 1.8 Update 152 (1.8.0_152). It is recommended that you downgrade back to 1.8.0_151.
---
+** Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended). However, if you run a Standalone instance of Mule and Anypoint Studio freezes during start up, check to see if your installed JDK runtime is 1.8 Update 152 (1.8.0_152). If it is, downgrade to 1.8.0_151 to correct the problem.
 ** IBM JDK 1.8
 ** OpenJDK 8 (JDK 1.8.0_162 recommended)
 +
-* For Microsoft Dynamics GP connector install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
-
-Mule Runtime runs as a standalone server on operating systems that support one of the required Java versions. An application server is not required; however, Mule Runtime can run under one. +
-Mule Runtime does not require a database unless you need to access a data store.
-
-[%header,cols="20a,75a"]
-|===
-|Software |Version
-|Operating systems |MacOS 10.11.4, HP-UX 11i V3, AIX 7.2, Windows 2012 R2 Server, Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 15.04 and 16.04, any other Linux distribution which runs a supported JDK and Linux kernel 3.13+
-|Application servers |Tomcat 7 and 8, WebLogic 12c, WebSphere 8, WildFly 8 and 9, Jetty 8 and 9
-|Databases |Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014
-|Browsers for working on Anypoint Platform | Firefox or Chrome (latest), Internet Explorer 10 or later; minimum screen resolution of 1024x768
-|===
\ No newline at end of file
+* An operating system that supports one of the supported Java versions:
++
+** MacOS 10.11.4
+** HP-UX 11i V3
+** AIX 7.2
+** Windows 2016, Windows 2012 R2 Server, or Windows 8.1
+** Solaris 11.3
+** RHEL 7
+** Ubuntu Server 15.04 or 16.04, or any other Linux distribution which runs a supported JDK and Linux kernel 3.13
+* An application server is not required because Mule runs as a standalone server. However, Mule Runtime can run under one of these supported application servers:
++
+** Tomcat 7 and 8
+** WebLogic 12c
+** WebSphere 8
+** WildFly 8 or 9
+** Jetty 8 or 9
+* If you use the Microsoft Dynamics GP connector, install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
+* If you need to access a data store, a database is required. Mule Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014.
+* Supported browsers with a minimum screen resolution of 1024x768 (for working with the Anypoint Platform):
++
+** Latest Firefox version 
+** Latest Chrome version
+** Internet Explorer 10 or later
```

Comment:
Noam Ben Ami says that IE 11 isn't supported by Anypoint Platform: "Noam Ben-Ami [3:57 PM]
This is a rather common point of confusion and it would be good if the docs team can work to clarify this. The above link references the _mule-runtime_ docs, as far as I can tell. That's a different "thing" than Anypoint Platform, which does not support IE11. HTH."  It's not clear why we talk about Anypoint Platform here anyway because the section is on the Mule Standalone, not Anypoint Platform.

Reply:
I was wondering that too! Will replace this and the sentence about RN with See Also links.